style.css

```diff
@@ -47,11 +46,12 @@ body {
 }
 
 .work {
-  text-align: center;
-  background-image: url(fon_section-with-man.jpg);
+  background-image: url(./img/with-man.jpg);
```

Comment:
I can't see these changes in `style.css`. It looks like you forgot to run `npm run sass`. Please make sure you have changes in `style.css` syncronized with `style.scss`, and check how it works in the browser.